Sources/CorePayments/Networking/APIClient.swift

```diff
@@ -59,6 +59,7 @@ public class APIClient {
                 
         // TODO: - Move JSON encoding into custom class
         let postBody = GraphQLHTTPPostBody(query: request.query, variables: request.variables)
+        // TODO: - encoding `Data` results in mumbo jumbo string. Why
         let postData = try JSONEncoder().encode(postBody)
```

Comment:
interess: what ends up happening to the String?

Reply:
Woops, this is an outdated comment! Removed in a467c992ab1637a07ba0ec7eae5f630d35beea78